internal/bus/accounts.go

```diff
@@ -16,45 +16,125 @@ import (
 	"lukechampine.com/frand"
 )
 
-var errAccountsNotFound = errors.New("account doesn't exist")
+var (
+	ErrAccountNotFound = errors.New("account doesn't exist")
+)
 
-type accounts struct {
-	mu     sync.Mutex
-	byID   map[rhpv3.Account]*account
-	logger *zap.SugaredLogger
-}
+type (
+	EphemeralAccountStore interface {
```

Comment:
This is the interface satisfied by the sql store right? So it's not ephemeral?

Reply:
Haha ok. I'll change it.